hknweb/candidate/views.py

```diff
@@ -255,7 +279,7 @@ def sort_rsvps_into_events(rsvps):
 
 # Checks which requirements have been fulfilled by a candidate
 def check_requirements(sorted_rsvps):
-    #TODO: increase flexibility by fetching event requirement count from database
+    # TODO: increase flexibility by fetching event requirement count from database
```

Comment:
Probably be easier to track TODOs like these if you file issues for these instead of waiting for someone to grep for these.

Reply:
There are so many issues... though is one in particular does seem like a long term investment

Reply:
What if we have both? An issue so we know it's there, and a TODO so you know where in the code to look. Can't hurt

Reply:
So issues can reference where to look in the code, too: https://github.com/compserv/hknweb/blob/0306ddcb3a6fa7ae3c984dc391eb8a9a7e5d2738/hknweb/candidate/views.py#L282

Reply:
But ok, fine, you can leave the TODO as long as it's accompanied by an issue.